api/models/templates/app.tmpl

```diff
@@ -332,7 +332,11 @@
         "SecurityGroupIngress": [
           {{ range .PortMappings }}
             {
+              {{ if .Public }}
               "CidrIp": "0.0.0.0/0",
+              {{ else }}
+              "CidrIp": "10.0.0.0/16",
```

Comment:
hard coding this is incorrect. We can get VPCCIDR from the rack environment, which will reflect whatever parameter value was used when installing convox.